content/pl/docs/reference/glossary/index.md

```diff
@@ -0,0 +1,15 @@
+---
+approvers:
+- chenopis
+- abiogenesis-now
```

Comment:
Can you remove these lines? We want to use the reviewers/approvers from the OWNERS file instead. Only when the localization will be having a lot of people, files can be appointed to specific contributors.